.github/workflows/continuous_integration.yml

```
@@ -10,32 +10,37 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repo
-        uses: actions/checkout@v4
+        uses: actions/checkout@v6
         with:
           fetch-depth: 0
       - name: Fetching node version
         run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
```

actionlint (1.7.9)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)